net/fvpn/Makefile

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: MIT
+#
+# Copyright (C) ForestVPN.com
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fvpn
+PKG_VERSION:=0.2.1-post.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```

Comment:
Oh, this is wrong. The tarball will be downloaded as "v$(PKG_VERSION).tar.gz" in dl folder w/o any name.